src/primevue/toast/toast.ts

```diff
@@ -0,0 +1,62 @@
+import { ToastPassThroughOptions } from "primevue/toast";
+import { tw } from "@/lib/tags.ts";
+
+const toast: ToastPassThroughOptions = {
+    message: ({ props }) => {
```

Comment:
Todo: when stacking several toasts there should be a gap between them. Also the internal layout seems broken:

![Bildschirmfoto 2024-09-06 um 18 01 36](https://github.com/user-attachments/assets/575c1d01-d1f9-470e-ad9a-c4c2ea9dccc2)